bumble/crypto.py

```diff
@@ -100,6 +100,49 @@ def dh(self, public_key_x: bytes, public_key_y: bytes) -> bytes:
 # -----------------------------------------------------------------------------
 
 
+# -----------------------------------------------------------------------------
+def generate_prand() -> bytes:
+    '''Generates random 3 bytes, with the 2 most significant bits of 0b01.
+
+    See Bluetooth spec, Vol 6, Part E - Table 1.2.
+    '''
+    prand_bytes = secrets.token_bytes(6)
+    return prand_bytes[:2] + bytes([(prand_bytes[2] & 0b01111111) | 0b01000000])
+
+
+# -----------------------------------------------------------------------------
+def generate_random_static_address() -> bytes:
```

Comment:
Maybe this function should return an `Address`, with the right address type? (Also, this is a function that *uses* crypto, but isn't a crypto function itself, so maybe it should live somewhere else?

Reply:
I think rand / prand is also considered crypto somehow? But yes, they're better to be classmethods of `Address`. The only exception is prand - hosting it in hci/Address is not very proper, but there isn't a module more proper than crypto(from my point of view).

Reply:
Moving address methods to https://github.com/google/bumble/pull/379